app/pages/AppContainer.spec.js

```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import React from 'react';
+import DocumentTitle from 'react-document-title';
+import simple from 'simple-mock';
+
+import { UnconnectedAppContainer as AppContainer } from './AppContainer';
+
+describe('pages/AppContainer', () => {
+  function getWrapper(props) {
+    const defaults = {
+      fetchUnits: simple.mock,
+    };
+    return shallow(<AppContainer {...defaults} {...props} />);
+  }
+
+  describe('render', () => {
+    describe('DocumentTitle', () => {
+      it('is rendered', () => {
+        const title = getWrapper().find(DocumentTitle);
+        expect(title.length).to.equal(1);
+      });
+
+      it('gets correct title prop', () => {
+        const title = getWrapper().find(DocumentTitle);
+        expect(title.prop('title')).to.equal('Huonevarausjärjestelmä');
+      });
+    });
+
+    it('renders a div with className "app"', () => {
+      const div = getWrapper().find('div');
+      expect(div.length).to.equal(1);
+      expect(div.prop('className')).to.equal('app');
+    });
+  });
```

Comment:
- [ ] Maybe this could include a tests that check that a `<AppContainer>Some children</AppContainer>` renders the children